hsds/basenode.py

```diff
@@ -134,10 +148,10 @@ def getHeadUrl(app):
     try:
         await http_post(app, req, data=body)
     except ClientError as client_exception:
-        log.error(f"got ClientError registering with oio_proxy: {client_exception}")
+        log.error(f"got ClientError registering with oio_proxy: {client_exception} and body {body}")
         return
     except CancelledError as cancelled_exception:
-        log.error(f"got CanceeledError registering with oio_proxy: {cancelled_exception}")
```

Comment:
Please change "Canceeled" to "Cancelled"!  (bad spelling in the original code I know)

Reply:
fixed